admin/kernel-masterdata-service/src/test/java/io/mosip/kernel/masterdata/test/service/SchemaServiceTest.java

```diff
@@ -328,4 +328,11 @@ public void testDeleteIdentitySchemaFailedUpdate() throws Exception {
 		identitySchemaService.deleteSchema("test-test");
 	}
 
+	@Test
+	@WithUserDetails("reg-officer")
+	public void testFetchAllDynamicFieldsAllLang() throws Exception {
```

Comment:
Why is dynamic field test case part of the Schema Service Test class?

Reply:
Moved to master data service class